### ISSUE TITLE
What is needed for STM32F4xx OTP support?

### ISSUE THREAD

**Issue by anthrax-0** [mbedmicro/pyOCD#761]
Hi,
It seems that builtin targets for stm32f4xx do not have support for OTP area and that .FLM for it is present in the Keil pack and is the same for all of them.
 Is it enough to just add additional FLASH_ALGO to the target and use it in the corresponding FlashRegion or there are some other changes required?

**Comment by flit** [comment]
Yes, that should be all that is required. Just generate the flash algo blob as described in [adding_new_targets.md](https://github.com/mbedmicro/pyOCD/blob/master/docs/adding_new_targets.md), insert the generated `FLASH_ALGO` dict, and add an appropriate `FlashRegion` to the target's memory map.

You might want to add a custom flash class with an `override_security_bits()` method, like `pyocd.target.family.flash_kinetis.Flash_Kinetis`, that prevents accidental locking of the device.
